deployment/nginx.conf

```diff
@@ -0,0 +1,27 @@
+worker_processes 1;
+
+error_log stderr;
+
+events {
+    worker_connections 1024;
+}
+
+http {
+    include mime.types;
+    default_type application/octet-stream;
+
+    server {
+        listen 8080 default;
+        client_max_body_size 1m;
+        server_name _;
+
+        keepalive_timeout 65;
+
+        root   /opt/frontigrade/build;
```

Comment:
still seeing a reference to frontigrade here - is this right?

Reply:
good catch